src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC, memo } from 'react';
+import { Good } from '../../types/Good';
+
+function areGoodsEqual(prevGoods: Good[], currGoods: Good[]): boolean {
```

Comment:
it's better to create `utils` folder and separate `areGoodsEqual.js` file for this function